config/locales/zh.yml

```diff
@@ -1492,6 +1492,7 @@ time.formats.event_timestamp: '%B %-d, %Y at %-l:%M %p'
 time.formats.event_timestamp_js: '%{year} %{month} %{day}, %{hour}:%{minute} %{day_period}'
 time.formats.event_timestamp_utc: '%B %-d, %Y， %-l:%M %p UTC'
 time.formats.event_timestamp_with_zone: '%B %-d, %Y，%-l:%M %p %Z'
+time.formats.full_date: '%Y 年 %B %-d 日'
```

Comment:
I *think* I see now, `full_date` doesn't exist for other locales. I think we should add it, so the template can use `full_date` across all locales. The other locales can copy their values from `event_date` since that appears to be appropriate?

Reply:
ahh i think i understand - `full_date` for the other translations should have the same value as `event_date`?